src/internal/operators/timeInterval.ts

```diff
@@ -23,8 +23,7 @@ import { map } from './map';
  * Emit interval between current value with the last value
  *
  * ```ts
- * import { interval } from "rxjs";
- * import { timeInterval, timeout } from "rxjs/operators";
+ * import { interval, timeInterval } from 'rxjs';
```

Comment:
I dismissed `seconds.pipe(timeout(900))...` example from lines 36 through 40 in one of the later commits, this is why I don't have import for `timeout` here.